### DISCUSSION TITLE
Assigning div classes on the fly

### DISCUSSION THREAD

**Discussion by craigbass76** [squidfunk/mkdocs-material#4653] [Q&A] [answered]
```
:::defs

- First
    - blah
- Second
    - lahblah
- Third
    - blahblahblah

:::
```
I'm using pandoc (which I've just learned has it's own flavor of markdown) and it renders the above code as:
```
<div class="defs">
<ul>
<li>First
<ul>
<li>blah</li>
</ul></li>
<li>Second
<ul>
<li>lahblah</li>
</ul></li>
<li>Third
<ul>
<li>blahblahblah</li>
</ul></li>
</ul>
</div>
```

I guess MkDocs uses python-flavored Markdown. Here's what my source looks like in MkDocs:
```
<p>:::defs</p>
<ul>
<li>First<ul>
<li>blah</li>
</ul>
</li>
<li>Second<ul>
<li>lahblah</li>
</ul>
</li>
<li>Third<ul>
<li>blahblahblah</li>
</ul>
</li>
</ul>
```

Is there any quick way to create a div with a class, besides hand coding the HTML? I'm trying to do two different things with the same `.md` files.

**Comment by facelessuser** [comment, marked as answer]
I'm just going to link back to here instead of repeating myself: https://github.com/mkdocs/mkdocs/discussions/3042#discussioncomment-4161377

**Reply by squidfunk** [reply]
@craigbass76 please don't double post. You're doubling work for us maintainers.

**Reply by craigbass76** [reply]
I was aiming for the other repo initially and didn't realize I was here. Sorry about that.
